### PR TITLE
[TEAM1] Add Structure section and team 1 README

### PR DESCRIPTION
**Pull request by Darrekt**
# Summary

Please provide a summary of the change.

## Additional Information

Auxiliary information goes here.

## Test Plan

- Add information on how you tested your changes.

- `go test ./...`
```
Paste logs here
```

- `go run .`
```
Paste logs here
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,4 +18,21 @@ go build # build step
 ## Testing
 ```bash
 go test ./...
-```
\ No newline at end of file
+```
+
+## Structure
+
+### `Docs`
```
Comment (lhl2617) [inline README.md]: lowercase d

File: README.md [hunk 2]
```diff
@@ -18,4 +18,21 @@ go build # build step
...
+Important documents pertaining to codebase organisation, code conventions and project management. Read before writing code.
+
+### `internal`
+All code goes in here.
+
+#### `clients`
+ Individual team code goes into the respective folders in this directory.
+
+ #### `common`
```
Comment (lhl2617) [inline README.md]: whitespace?

**Review by lhl2617 — CHANGES_REQUESTED** [review]
add missing details + small nits in readme

File: internal/clients/team1/README.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Roles of this Agent
+TBC
+
+# Team Members
+
+| Rep? | Name      | Shortcode | Github ID |
+| - | --------- | --------- | --------- |
+| Design | Naim Govani | ng2517 | NGovani |
+| Infrastructure | Lee Lin Hao (LH) | lhl2617 | lhl2617 |
```
Comment (lhl2617) [inline internal/clients/team1/README.md]: `Lin Hao Lee` preferred but this is fine 👍

**Review by Darrekt — COMMENTED** [review]
Fixed all requests

File: README.md (hunk 2 above)
Comment (lhl2617) [reply README.md]: White space between LHS and ####?

**Review by lhl2617 — APPROVED** [review]
Preapproving--pls merge after fixing white space :)

**Pull request merged**